### PR TITLE
Update server_readymulti.md

### PR DESCRIPTION
**Pull request by Sup3rFire**

### CODE REVIEW

File: Messages/server_readymulti.md [hunk 1]
```diff
@@ -8,6 +8,33 @@ A message indicating that the custom room is about to start playing.
     * (integer) `id`: [The id of this message.](../Ribbon.md#id-messages)
     * (string) `command`: Always `"readymulti"`.
     * (object) `data`:
+        * (object) `options`:
```
Comment (Sup3rFire) [inline Messages/server_readymulti.md:11]: This was taken from the [gmupdate](https://github.com/Poyo-SSB/tetrio-bot-docs/blob/master/Messages/server_gmupdate.md) event they seem to be using the same values.

**Comment by Poyo-SSB** [comment]
In a case like this I think it would be better to separate this object into its own "Options" type.

**Comment by Sup3rFire** [comment]
> In a case like this I think it would be better to separate this object into its own "Options" type.

I think we should just add a new section to the room settings file for the inaccessible settings.

**Comment by Sup3rFire** [comment]
> In a case like this I think it would be better to separate this object into its own "Options" type.

I have added a section for it in [this](https://github.com/Poyo-SSB/tetrio-bot-docs/pull/19) PR.

**Pull request merged**